lua/telescope/actions/init.lua

```diff
@@ -1017,7 +1017,11 @@ end
 actions.delete_buffer = function(prompt_bufnr)
   local current_picker = action_state.get_current_picker(prompt_bufnr)
   current_picker:delete_selection(function(selection)
-    vim.api.nvim_buf_delete(selection.bufnr, { force = false })
+    if vim.api.nvim_buf_get_option(selection.bufnr, "buftype") == "terminal" then
+      vim.api.nvim_buf_delete(selection.bufnr, { force = true })
```

Comment:
how about

```lua
local force = vim.api.nvim_buf_get_option(selection.bufnr, "buftype") == "terminal"
vim.api.nvim_buf_delete(selection.bufnr, { force = force })
```

Reply:
Sure, don't have a strong opinion on this. Will change it.